doc/DOWNLOADS.md

```diff
@@ -105,14 +105,21 @@ In order to upgrade Nicotine+ to a newer release, run the following:
 scoop update nicotine-plus
 ```
 
-### Chocolatey
+### [Chocolatey](https://chocolatey.org/install)
 
-Nicotine+ can be installed using [Chocolatey](https://community.chocolatey.org/packages/nicotine-plus). Run the following:
+Nicotine+ can be installed using [Chocolatey](https://community.chocolatey.org/packages/nicotine-plus). Run the following command:
 
 ```sh
 choco install nicotine-plus
 ```
 
+To upgrade Nicotine+ with chocolatey, run the following:
```

Comment:
If the above removed link to the Nicotine+ community Chocolatey page is desired, perhaps it can be added here: 

```
To upgrade [Nicotine+ using chocolatey](https://community.chocolatey.org/packages/nicotine-plus), run the following:
```
 To upgrade [Nicotine+ using chocolatey](https://community.chocolatey.org/packages/nicotine-plus), run the following: